models/DataType.yaml

```diff
@@ -1,7 +1,8 @@
 keyword: String  # Keyword that identifies the type i.e. int, str
 # Integer attributes
-integer-min: String  # String representing minium integer or uncertain
+integer-min: String  # String representing minimum integer or uncertain
 integer-max: String  # String represetinng maximum integer or uncertain
 integer-signed: Boolean  # Whether the datatype is signed integer or not
 delimiters: Delimiter[] # If the datatype has any delimiters
 literals: Keyword[] # List of literals for a datatype
+precision: Integer # Integer representing precision of floating point data types.
```

Comment:
two spaces before `#`

(This repo has a .coafile now)

Reply:
Done.